go.mod

```diff
@@ -36,7 +36,7 @@ require (
 // and https://github.com/Azure/go-autorest/issues/481
 replace (
 	github.com/Azure/go-autorest v11.1.2+incompatible => github.com/Azure/go-autorest v12.1.0+incompatible
-	k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
```

Comment:
I had to update to a newer version of client-go to get the AdmissionregistratioV1() API. It took me a minute to realize the numbers before the commit SHA are the commit date 🤦‍♂

Reply:
I learned a helpful command to help get this pseudo-version right. You might already know it. Once you have the commit SHA you want, you can run `go get k8s.io/client-go 1fbdaa4c8d90` and it'll fill out the pseudo-version for you. https://tip.golang.org/cmd/go/#hdr-Pseudo_versions

Reply:
Nice! Thanks